apex-app-archetype/src/main/resources/archetype-resources/src/test/resources/log4j.properties

```diff
@@ -1,4 +1,4 @@
-log4j.rootLogger=DEBUG,CONSOLE
+log4j.rootLogger=INFO,CONSOLE
```

Comment:
@tweise Thomas, can you please clarify. Do you mean that log level should not be changed from DEBUG to INFO at all or your comment applies only to archetype? Note that travis-ci builds already fail due to the limit, so we need to reduce amount of logging.

When users develop applications, do they want to see datatorrent and other 3rd party libraries debug level output? Should not that be limited to INFO level and they need to set DEBUG level for their own classes?